CPU/frame.py

```diff
@@ -0,0 +1,92 @@
+from tkinter import *
+from tkinter import ttk
+from turtle import width
+from typing_extensions import Self
+
+from integrator import frame as modelFrame
+from logger.level import LogLevel
+from logger.logger import Logger
+from . import graph
+from .cpu_data.global_cpu import GlobalCPU
+
+class Tab (modelFrame.DOSMFrame):
+
+    def __init__(self, master, logger: Logger, **options):
+        super().__init__(master, logger, **options)
+        self.cpu = GlobalCPU()
+        self.dataTree = ttk.Treeview()
+        self.cpuUsageGraph = graph.LineGraph(self)
+
+    def show(self):   
+        # Frame with one row and two columns
+        self.columnconfigure(0, weight=1)
+        self.columnconfigure(2, weight=1)
+        self.rowconfigure(0, weight=1)
+
+        # data tree
+        self.generateTreeView(self)
+        self.dataTree.grid(row=0, column=0, columnspan=1, sticky=N+S+E+W)
+
+        # graph Frame
+        graphFrame = Frame(self, width=100, height=10)
+        graphFrame.grid(row=0, column=2, sticky=N+S+E+W)
+
+
+        #graph data
+        self.cpuUsageGraph = graph.LineGraph(graphFrame,width=40, padx=40, pady=40)
+        self.cpuUsageGraph.pack(fill=BOTH)
+        self.cpuUsageGraph.show()
+
+
+        
+    def update(self, dt):
```

Comment:
Je crois que tu as oublié d'appeler la méthode `stats.update()` et `time.update()`. Corrige-moi si je me trompe

Reply:
L'ingénieuse proposition formalisé de @RedbeanGit : 
```suggestion
    def update(self, dt):
       stats.update()
       time.update()
```

Reply:
Bien tenté mais ça ne marchera pas parce que `stats.update` prend un argument `stats` et `time.update` un argument `time`

Reply:
Elles sont updates au moment de l'update de global_cpu donc normalement c'est ok ;) (c'est lui qui gère et contient tout donc je le laisse se charger des sous-logiques ;) )

Reply:
Ah oui aux temps pour moi du coup problème réglé